src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstBase;
+    private int secondBase;
+    private int leg;
+
+    public IsoscelesTrapezoid(int firstBase, int secondBase, int leg, String color) {
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.leg = leg;
+        setColor(color);
```

Comment:
create a constructor in super class for this property, call this constructor instead of a setter